cmake/modules/gluecodium/gluecodium/Generate.cmake

```diff
@@ -103,10 +103,10 @@ function(apigen_generate)
   _apigen_set_option_or_default(VERSION +)
 
   set(operationVerb "Generate")
-  set(validateParam "")
+  set(validateProperty "false")
   if(${apigen_generate_VALIDATE_ONLY})
     set(operationVerb "Validate")
-    set(validateParam "-validateOnly")
+    set(validateProperty "true")
```

Comment:
What does this change mean?

Reply:
-validateOnly was command line flag to enable validation (with actual generation as i understand).
Its analog in property file is
validate=[true|false]

Reply:
Does this mean the command line option is changed? Where is the property file located and how is it specified?

Reply:
Command line options are moved to property file. Property file is created below and it's path is specified like
> set (_gluecodium_options_file "${CMAKE_CURRENT_BINARY_DIR}/gluecodium-options-${APIGEN_TARGET}.txt")

It's unique for each source folder + target. Almost all command line options are now stored there and path to file is passed to gluecodium with only one command line option -options=<path-to-file> in runGenerate.cmake

Reply:
OK. Ic. That's property file generated later.